README.md

```diff
@@ -64,14 +66,47 @@ function configureStore(initialState) {
 export default configureStore
 ```
 
-### Wrap Page Component
+### Configure Custom <App>
 
 ```js
-import React, {Component} from 'react'
+import React from 'react'
+import App, {Container} from 'next/app'
 import withRedux from 'next-redux-wrapper'
-import withReduxSaga from 'next-redux-saga'
+import nextReduxSaga from 'next-redux-saga'
 import configureStore from './configure-store'
 
+class ExampleApp extends App {
+  static async getInitialProps({Component, ctx}) {
+    let pageProps = {}
+
+    if (Component.getInitialProps) {
+      pageProps = await Component.getInitialProps(ctx)
+    }
+
+    return {pageProps}
+  }
+
+  render() {
+    const {Component, pageProps, store} = this.props
+    return (
+      <Container>
+        <Provider store={store}>
+          <Component {...pageProps} />
+        </Provider>
+      </Container>
+    )
+  }
+}
+
+export default withRedux(configureStore)(withReduxSaga(ExampleApp))
```

Comment:
`export default withRedux(configureStore)(nextReduxSaga(ExampleApp))`

isn't it?